src/led_strip.py

```diff
@@ -2,15 +2,58 @@
 from time import sleep
 import uasyncio as asyncio
 
+# :TODO: Unsure on naming of this mixin/class. Also once that's settled, move to another file.
+class LedGestures:
+    # set to a new color
+    async def until_color_changed(self, color): 
+        self.pixels.fill(color)
```

Comment:
Shouldn't this call `until_faded_to` to make it animated? This currently would just flip the color immediately. If that's what you intend, this method doesn't need to be async, and the `until` prefix implies otherwise.